### ISSUE TITLE
AOS is not defined

### ISSUE THREAD

**Issue by abbasidaniyal** [abbasidaniyal/PlasmaDonorFinder#37]
@zrthxn This comes up in the console. Is this significant?

**Comment by zrthxn** [comment]
Not really, it won't break anything useful. But it might be useful later so maybe keep the issue open as a reminder to fix it
